integration-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+EXPECTED_HASH="a000ca9241a286441bc46bfdb5c12723" 
+
+echo "Running sizer command..."
+./sizer.sh <(aws s3 cp s3://nao-testing/sequence-tools/sizer/raw/test01_1.fastq.gz - | gunzip) \
+           <(aws s3 cp s3://nao-testing/sequence-tools/sizer/raw/test01_2.fastq.gz - | gunzip) \
+           sizer_integration_test \
+           s3://nao-testing/sequence-tools/sizer/siz/test01
+
+# Wait briefly for S3 consistency
+sleep 1
+
+# md5sum will give us a trailing " -" since we provide stdin input, so we cut that off
```

Comment:
`md5sum --quiet` fixes this:

```
$ echo foo | md5sum
d3b07384d113edec49eaa6238ad5ff00  -
$ echo foo | md5sum --quiet
d3b07384d113edec49eaa6238ad5ff00
```

Reply:
Mmm, looks like this is in the version on mac, but not the modern `md5sum` that linux systems are likely to have.  Oh well.  What you have is good!